user-creation/src/main/java/no/unit/nva/useraccessservice/usercreation/person/cristin/model/CristinAffiliationInstitution.java

```diff
@@ -20,7 +22,7 @@ public String getId() {
         return id;
     }
 
-    public String getUrl() {
-        return url;
+    public URI getUri() {
```

Comment:
We apply some constraint to the value read from Cristin here. Not sure if I like a potential RuntimeException just trying to get the value we read from Cristin. But on the other hand failing fast is a modern pattern :)